src/eval_tool.py

```diff
@@ -0,0 +1,64 @@
+import cPickle
+import os
+import pandas as pd
+import numpy as np
+
+from keras.models import load_model
+from keras.preprocessing.text import Tokenizer
+from keras.preprocessing.sequence import pad_sequences
+from keras.utils import to_categorical
+from sklearn import metrics
+
+def compute_auc(y_true, y_pred):
+    fpr, tpr, thresholds = metrics.roc_curve(y_true, y_pred)
+    return metrics.auc(fpr, tpr)
+
+class EvalTool():
+    def __init__(self, 
+                 model_version = None, 
+                 model_dir = '../models'):
+        self.model_dir = model_dir
+        if model_version:
+            self.load_model_and_tokenizer(model_version)
```

Comment:
Is there a usecase for this class if we don't pass model_version?

I think we could just make model_version required. We could also inline `load_model_and_tokenizer` into `__init__`. (This is more inline with making this a `Model` class.)

Reply:
I just wanted to provide options, but you're right. I've simplified.